src/components/nudge/Nudge.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../styles/variables';
+
+/**************************************
+ * Nudge
+ **************************************/
+
+.Nudge {
+    position: fixed;
+    right: 20px;
+    bottom: 40px;
+    z-index: 40;
+    width: 221px;
+    max-height: 450px;
+    padding: 25px;
+    background-color: $white;
+    border: solid 1px $bdl-gray-02;
+    border-radius: 30px;
+    box-shadow: 0 3px 6px 0 rgba(0, 0, 0, .1);
+
+    &--isClosed {
+        display: none;
+    }
+
+    &-button button {
```

Comment:
with SUIT style, you shouldnt need to do nesting. So all these can move out.

.bdl-Nudge-closeButton { ... }

Reply:
Search for "CSS Class naming" in confluence.